tests/test_serializers/test_utils.py

```diff
@@ -218,6 +218,28 @@ def __init__(self):
         # Assert
         assert result == dict(a=1, b=2)
 
+    @staticmethod
+    def test_uninstantiated_class():
+        """ Test encoding for uninstantiated classes with a special skip function defined """
+
+        # Arrange
+        class UninstantiatedClass:
+            def __init__(self):
+                self.a = 1
+                self.b = 2
+                self.c = 3
+
+            @classmethod
+            def __snapshotskip__(cls):
+                encoding = "Dont Encode Me!"
+                return encoding
```

Comment:
This is interesting. It's not really _not_ encoding it, it's just overriding what the encoding is. Another interesting trick you can throw at this test case is including vs not including the `__snapshot__` method.

Reply:
[reverted](https://github.com/MORSECorp/snappiershot/pull/96/commits/5e743c9bcbaebe11b2c77c1050831607853a458d)